### PR TITLE
Translation Base_Dutch.lang

### PR DESCRIPTION
**Pull request by faab007nl**

### CODE REVIEW

File: translations/Base_Dutch.lang [hunk 1]
```diff
@@ -1,133 +1,133 @@
...
+update.updatedPlugin: "Plugin bijgewerkt naar **%**, herladen.."
+update.youAreNotVerified: "U bent niet geverifieerd op onze **Discord**"
+update.youNotOwnPlugin: "U bent niet de eigenaar van deze plug-in. **Koop het** om deze functie te ontgrendelen"
+updateView.no.action: "Klik om op deze versie te blijven"
+updateView.no.desc.newVersion: "Nieuwe versie: **%**"
+updateView.no.desc.yourVersion: "Jou versie: **%**"
+updateView.no.title: "Blijf op deze versie"
+updateView.title: "Bijwerken van %from% naar %to%"
+updateView.yes.action: "Klik om **Bijwerking** te laden"
+updateView.yes.desc: "Deze functie vereist ** Discord ** om uw aankoop te verifiëren"
```
Comment (rlnt) [inline translations/Base_Dutch.lang]: Is there a specific reason why you added spacing before and after the `Discord`?

Comment (faab007nl) [reply translations/Base_Dutch.lang]: Oh no that's ecedental

**Review by rlnt — APPROVED** [review]
Thanks for your contribution!

**Comment by faab007nl** [comment]
NP

**Pull request merged**